### ISSUE TITLE
An exception has been thrown during the rendering of a template ("Call to undefined function twig_get_attribute()").

### ISSUE THREAD

**Issue by stovesy** [yiisoft/yii2-twig#153]
After a recent composer/update, I'm getting the above error.

### What steps will reproduce the problem?
{{ this.registerCssFile("@web/css/report.css") }}

### What's expected?
No error
### What do you get instead?
An exception has been thrown during the rendering of a template ("Call to undefined function twig_get_attribute()").
### Additional info
Twig 3.9.0
Yii Framework 2 twig extension 2.4.2 September 04, 2022

Error is at line 54 of Template.php
return \twig_get_attribute($env, $source, $object, $item, $arguments, $type, $isDefinedTest, $ignoreStrictCheck);

**Comment by morten-thomassen** [comment]
This happened to me today. Downgrading twig has (temporarily) bypassed the issue:

`composer require "twig/twig:3.8.0"`

But obviously I would like to see a proper fix implemented

**Comment by diffy0712** [comment]
@schmunk42 @samdark 
I think https://github.com/yiisoft/yii2-twig/pull/155 released in `2.4.3` already fixed this issue, and once released https://github.com/yiisoft/yii2-twig/pull/154 in `2.5.0` will provide support for twig3.9 as well.
I think this issue is resolved.

**Comment by CasperHK** [comment]
I got the same issue.
```bash
$ composer show twig/twig
name     : twig/twig
descrip. : Twig, the flexible, fast, and secure template language for PHP
keywords : templating
versions : * v3.20.0
type     : library
license  : BSD 3-Clause "New" or "Revised" License (BSD-3-Clause) (OSI approved) https://spdx.org/licenses/BSD-3-Clause.html#lic
enseText
homepage : https://twig.symfony.com
source   : [git] https://github.com/twigphp/Twig.git 3468920399451a384bef53cf7996965f7cd40183
dist     : [zip] https://api.github.com/repos/twigphp/Twig/zipball/3468920399451a384bef53cf7996965f7cd40183 3468920399451a384bef
53cf7996965f7cd40183
path     : C:\xampp\htdocs\cctv_inventry\src\vendor\twig\twig
names    : twig/twig

support
issues : https://github.com/twigphp/Twig/issues
source : https://github.com/twigphp/Twig/tree/v3.20.0

autoload
files
psr-4
Twig\ => src/

requires
php >=8.1.0
symfony/deprecation-contracts ^2.5|^3
symfony/polyfill-ctype ^1.8
symfony/polyfill-mbstring ^1.3

requires (dev)
phpstan/phpstan ^2.0
psr/container ^1.0|^2.0
symfony/phpunit-bridge ^5.4.9|^6.4|^7.0

```
